pages.de/windows/xcopy.md

```diff
@@ -0,0 +1,36 @@
+# xcopy
+
+> Kopieren von Dateien und Verzeichnisbäumen.
+> Weitere Informationen: <https://learn.microsoft.com/windows-server/administration/windows-commands/xcopy>.
+
+- Kopieren der Datei(en) an den angegebenen Zielort:
+
+`xcopy {{pfad/zu/datei_oder_ordner}} {{pfad/zum/ziel}}`
```

Comment:
We use this translation in https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md
```suggestion
`xcopy {{pfad/zu/datei_oder_verzeichnis}} {{pfad/zum/ziel}}`
```

Reply:
I updated it :)